dissect/target/plugin.py

```diff
@@ -874,9 +1007,19 @@ def find_plugin_functions(
     a list of plugin function descriptors (including output types).
     """
     result = []
+
+    # Avoid cyclic import
+    from dissect.target.plugins.general.default import DefaultPlugin  # noqa
+    from dissect.target.plugins.general.example import ExamplePlugin  # noqa
+
+    def add_to_result(func: PluginFunction) -> None:
+        if func not in result and func.class_object not in [DefaultPlugin, ExamplePlugin]:
```

Comment:
Why not add another `__magic__` class variable? I.e. `__skip__`?

Reply:
I tried to avoid adding more class vars like that, but it seems inevitable.

Reply:
I agree that we should not keep adding infinite amounts of variables, but it seems cleaner than doing inline imports in this case 😄